content/numpy/concepts/math-methods/terms/ceil/ceil.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: '.ceil()'
+Description: 'Rounds each element in an array to the nearest greater integer.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'Math'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+In NumPy, the **`.ceil()`** function rounds each element in an array to the nearest integer that is greater than or equal to the element.
+
+This function is applied to real numbers.
+
+## Syntax
+
+```pseudo
+numpy.ceil(input_value, out=None, where=True)
+```
+
+- `input_value`: The input array or scalar that contains the elements to be rounded up.
+- `out` (optional): A location where the result of the rounding will be stored. If no value is provided, the default is `None`, and a new array is returned.
+- `where` (optional): A boolean array that specifies the condition for rounding:
+	- If the condition is `True` at a given position, the element will be rounded.
+	- If the condition is `False`, the original value is retained.
+	- If no value is provided, the `.ceil()` function is applied to all elements.
```

Comment:
```suggestion
- `input_value`: The input array or scalar that contains the elements to be rounded up.
- `out` (optional): A location where the result of the rounding will be stored. If no value is provided, the default is `None`, and a new array is returned.
- `where` (optional): A boolean array that specifies the condition for rounding:
	- If the condition is `True` at a given position, the element will be rounded.
	- If the condition is `False`, the original value is retained.
	- If no value is provided, the `.ceil()` function is applied to all elements.
```
The `where` and `out` parameters are not applicable here.

Reply:
According to the [numpy docs](https://numpy.org/doc/2.0/reference/generated/numpy.ceil.html) the parameters are optional but applicable.